doc/tutorials/constant_pH/constant_pH.ipynb

```diff
@@ -4,7 +4,20 @@
    "cell_type": "markdown",
```

Comment:
<p>In the first paragraph I would add "particle size \sigma <strong>in nm </strong>..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/4262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
@davidbbeyer I have not yet cleaned up the text. Modifications to the python code are mostly done but I only did those changes in the text that were directly related to changes in the code.

Reply:
<p>@davidbeyer: I do not understand, where would you like to add "particle size \sigma <strong>in nm"</strong> and why? In the current implementation, you can set particle size in any unit of length and pint takes care of converting it.</p>

Reply:
<p>I'm okay with leaving it out, but then you should also remove the reference to Kelvin ("We start by setting the desired temperature&nbsp;T in Kelvin") since it is an arbitrary unit as well.</p>

Reply:
<p>OK, now I see the point. Right, we should remove the Kelvin from there.</p>